### ISSUE TITLE
Provider is tracking an old, deprecated upstream project

### ISSUE THREAD

**Issue by Sodki** [pulumi/pulumi-okta#53]
Hi,

This provider seems to be tracking an old upstream project. The current official Terraform Okta provider seems to now be https://github.com/oktadeveloper/terraform-provider-okta and the latest version is 3.5.0.

Thanks

**Comment by stack72** [comment]
I will get this upgraded first thing tomorrow!

**Comment by lukemundy-vgw** [comment]
came here to post the same thing! went to start using this today but noticed it was out of date and missing some stuff.

**Comment by stack72** [comment]
Working on this as we speak! It will be all upgraded and released in a couple of hours

**Comment by Sodki** [comment]
I must say you guys are amazing in maintaining this project. Huge thank you!

**Comment by stack72** [comment]
Sorry it took longer than normal to get this out - I was waiting on a review :D

**Comment by mj1618** [comment]
Struggling to find a ServerPolicyRule i.e. https://registry.terraform.io/providers/oktadeveloper/okta/latest/docs/resources/auth_server_policy_rule

Should that have come across with this change?

**Comment by Sodki** [comment]
@mj1618 Not sure what's going on here, but this seems to be it: https://www.pulumi.com/docs/reference/pkg/okta/auth/serverpolicyclaim/

Maybe you can open a separate ticket to tackle the naming issue.

**Comment by stack72** [comment]
I am changing it as we speak @Sodki / @mj1618

**Comment by stack72** [comment]
Hi @Sodki / @mj1618 

when this release has finished the newly renamed module will be available - https://github.com/pulumi/pulumi-okta/actions/runs/302141222

Paul
